### ISSUE TITLE
[Feature Request] Follow On

### ISSUE THREAD

**Issue by cheesegrits** [Vavassor/Tusky#517]
First, thank you for all the work on adding multiple instances. Much appreciated.

However ... one thing missing is any way to follow a user from / on another instance. My use case is that I have half a dozen different accounts, but I do all my following on my "home" instance. Although it's clunky, in Tootdon I can use "copy URL" from the hamburger menu, switch to my home instance, search and follow. But in Tusky, unless I'm missing something, I can't even copy the URL.

What would be peachy keen would be a a (perhaps optional) extra step on Follow, with a drop down of instances, to select the one to add the follow on. Or simpler, perhaps just a setting to specify whether to always use a specified instance to follow on, or use normal behavior. 

If that's not feasible, just a copy URL menu item would be nice.

**Comment by charlag** [comment]
Hi, thanks for reporting! I think that this is totally a valid use case, multi-account is by no means finished, it just have minimum functionality for now.
I am not sure what's the best way to do this, perhaps we could run a poll on UI choices.

**Comment by cheesegrits** [comment]
Thanks for the consideration. I don't really know how other  people use multiple accounts. Personally, I try and keep all my interactions to my "home" account, so a global "do everything from home account" switch would be useful - any favoriting, boosting, replying etc. I really just use the secondary accounts to get a different view onto the fediverse. But just the ability to "follow on" (a slightly less clunky version of the web based "remote follow") would be enough.

**Comment by novadeviator** [comment]
use cases with multiple accounts are different. 

i think it would be wrong to assing one as default (at least as the only option). sometimes you might see a toot to which you want to reply (or profile you want to follow) with a different account - some people keep different accounts for different topics. 

so it would be better to have a flexibility to be able to choose an account everytime you do certain action.

**Comment by tenkabuto** [comment]
I propose that the UI option for this could be hidden behind a long-press of each button. Upon long-press, an account selection prompt appears.

With this long-press, the action that a button would perform would then change from "Perform [action, such as Fav/Boost] as [current user]" to "Perform [action] as [list of all accounts configured in Tusky]".

**Comment by charlag** [comment]
Yeah, I see it in kinda same way

**Comment by charlag** [comment]
I 100% agree that we need this. Just no hands right now. It's relatively simple task and if someone stepped in it would be wonderful

**Comment by nupanick** [comment]
I agree that the natural way to do this would be a long-press on an interact button -- as a casual user that's where I instinctively looked for it before coming here.

**Comment by fentras** [comment]
And I'm in the same boat. Like @tenkabuto said — the simple but very useful feature would be exactly a long tap on _Follow_ button which opens popup (or fullscreen) menu to select desired account to follow from without switching from the active one.

**Popup Menu Example**

<a href="https://user-images.githubusercontent.com/26721650/69916291-657cfb80-146a-11ea-94d2-f7c2783315d9.png"><img src="https://user-images.githubusercontent.com/26721650/69916291-657cfb80-146a-11ea-94d2-f7c2783315d9.png" alt="Popup menu showing below Follow button with desired instance to select from" width="50%"/></a>

**Comment by JuanjoSalvador** [comment]
Is there any plans for this feature?

**Comment by bluebirch** [comment]
Opening profiles with another account works, so maybe that one can be checked in the list? Came here looking for feature requests regarding the same thing with toots.

**Comment by lukaprincic** [comment]
yes, seems like the state of things is:

- [x] Opening toots with another account 
- [x] Opening profiles with another account
- [ ] Following using another account
- [ ] boosting/faving using another account
- [ ] replying using another account

I wonder if 'following another account' needs traverse into that account, as boosts/favs should be done without leaving the timeline - the modal is already there for sharing from another app, right? every replying should just be a compose page where after the reply or cancel you'd be brought back where you were in an account where you were.. is this possible?

**Comment by charlag** [comment]
Opening things and composing is generally simple, we just switch to another account completely

**Comment by lukaprincic** [comment]
I'm grateful that currently it is possible to open posts with other account.  It's a great feature! However it has an unfortunate side-effect, it forgets your scroll-position in original timeline. That's why quick actions that would not take user away from where they were on the reading timeline would be quite amazing. Is this difficult to implement?
